tests/examples/40_job_template.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
@bzwei this file should get renamed to 46_job_template.yml also would we be running this example under Travis. It might cause test name conflicts in tests/test_examples.py if someone creates a test called test_46_something.